### PR TITLE
Redux saga

### PR DESCRIPTION
**Pull request by codeBelt**

### CODE REVIEW

File: src/app/components/home/home.component.html [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
-        <h2><a target="_blank" rel="noopener" href="https://github.com/angular/angular-cli/wiki">CLI Documentation</a>
-        </h2>
-    </li>
-    <li>
-        <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
-    </li>
-</ul>
+<div>
+    <div class="jumbotron">
+        <h1 class="display-3">{{(userReducer | async).name.title}} {{(userReducer | async).name.first}} {{(userReducer |
```
Comment (codeBelt) [inline src/app/components/home/home.component.html]: @n8rzz is there a better way to do this `{(userReducer | async).name.title}}`

File: src/app/stores/user/UserSaga.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+export class UserSaga {
+
+    public static * loadUser(action: IAction<void> = null) {
+        yield put({
+            type: LoadingAction.SET_LOADING,
+            payload: true,
+        });
+
+        const response: Response = yield fetch('https://randomuser.me/api/?inc=picture,name,email,phone,id,dob');
```
Comment (codeBelt) [inline src/app/stores/user/UserSaga.ts]: @n8rzz I should probably use Angular `http` but why shouldn't we just use `fetch`?

File: src/app/components/home/home.component.html (hunk 1 above)
Comment (codeBelt) [reply src/app/components/home/home.component.html]: Never mind I figured out a better way
`@select(['userReducer', 'name', 'first']) public readonly firstName: Observable<string>;`

Then I can do:
`{{(firstName | async)}}`

Comment (n8rzz) [reply src/app/components/home/home.component.html]: Ha. Well, I _was_ going to say try to move that logic to the component class. And that’s what you did here, nice work!

File: src/app/stores/user/UserSaga.ts (hunk 2 above)
Comment (n8rzz) [reply src/app/stores/user/UserSaga.ts]: A few questions there:
- Is fetch available in browser?
- is Angular.http a wrapper for fetch?

Comment (n8rzz) [reply src/app/stores/user/UserSaga.ts]: Also, what is the advantage to using fetch vs http?

**Pull request merged**